src/library/validators.js

```diff
@@ -2,7 +2,7 @@ const isName = str => /^$|^[A-zÀ-ú\- ]+$/.test(str)
 
 const isUsername = str => /^[\w!?$#@()\-*]+$/.test(str)
 
-const isPassword = str => /^([a-zA-Z\d!?$#@()\-*])+$/.test(str)
+const isPassword = str => /^([a-zA-Z\d!?$#%@()\\\-*/@^+*&:<>:{};~'.|[\]])+$/.test(str)
```

Comment:
you can rewrite this using two tests: the first one with `a-zA-Z\d` && the result of  `containSpecialCharacter()`

Reply:
for instance:
```js
/[a-zA-Z]/.test(str) && containSpecialCharacter(str)
```